src/main/kotlin/no/nav/syfo/dialogmote/database/MotedeltakerQuery.kt

```diff
@@ -19,13 +19,15 @@ const val queryCreateMotedeltakerArbeidstaker =
         created_at,
         updated_at,
         mote_id,
-        personident) VALUES (DEFAULT, ?, ?, ?, ?, ?) RETURNING id
+        personident,
+        fritekst_innkalling) VALUES (DEFAULT, ?, ?, ?, ?, ?, ?) RETURNING id
```

Comment:
Fritekster er relatert til varsler, i dette tilfelle så er en evt fritekst_innkalling noe som trenger å knyttes opp mot den konkrete innkalling som kan sendes til den konkrete deltaker.
For arbeidstaker så er det "ArbeidtakerVarsel"-objektet/tabellen som representerer et varsel som kan være av typen innkalling.

TIlsvarende løype er ikke laget for Arbeidsgiver ennå med varsel.